modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFolderMVCRenderCommand.java

```diff
@@ -31,6 +48,55 @@
 )
 public class CopyFolderMVCRenderCommand extends BaseFolderMVCRenderCommand {
 
+	@Override
+	public String render(
+			RenderRequest renderRequest, RenderResponse renderResponse)
+		throws PortletException {
+
+		try {
```

Comment:
Don't nest `try` blocks like this. Use a single try/catch block for both exceptions.